mkdocs.yml

```
@@ -104,7 +104,8 @@ nav:
       - Extras:
           - extras/intro.md
           - extras/fastapi.md
-          - extras/websocket.md
+          - extras/websocket_client.md
```

```suggestion
          - extras/websocket.md
```
The file is named websocket, not websocket_client

---

fixed the name of the file instead